### ISSUE TITLE
Refactor all generators to fix C++ indentation and generate Python code

### ISSUE THREAD

**Issue by KeyWorksRW** [KeyWorksRW/wxUiEditor#862]
### Description:

tldr; -- C++ indentation is currently incorrect, though code will still compile correctly. Do _not_ assume the Python code is correct.

Currently C++ code generation has the wrong indentation for a lot of the code. Rather than fix just that and then fix it again when code generation is switched over to use the new `Code` class, the current build will have indentation wrong until _all_ generators have been updated. This issue is pinned in order to track that work.

At this point the underlying code is sufficiently complete that it should be possible to generate Python code for all generators where there is a matching wxPython control. Once all code generation is in place, testing can start to see what changes need to be made to get wxPython to accept the code.
